src/storage/src/hummock/shared_buffer/shared_buffer_uploader.rs

```diff
@@ -169,14 +169,15 @@ impl SharedBufferUploader {
 
         let uploaded_sst_info: Vec<SstableInfo> = tables
             .iter()
-            .map(|sst| SstableInfo {
+            .map(|(sst, vnode_bitmap)| SstableInfo {
                 id: sst.id,
                 key_range: Some(risingwave_pb::hummock::KeyRange {
                     left: sst.meta.smallest_key.clone(),
                     right: sst.meta.largest_key.clone(),
                     inf: false,
                 }),
                 file_size: sst.meta.estimated_size as u64,
+                vnode_bitmap: vnode_bitmap.clone(),
```

Comment:
can we use into_iter and avoid the clone?

Reply:
thanks. #2230